modules/installing-python-packages-on-your-code-server-workbench.adoc

```diff
@@ -0,0 +1,70 @@
+:_module-type: PROCEDURE
+
+[id="installing-python-packages-on-your-code-server-workbench_{context}"]
+= Installing Python packages on your code-server workbench
+
+[role='_abstract']
+You can install Python packages that are not part of the default workbench image by adding the package and the version to a `requirements.txt` file and then running the `pip install` command in a terminal window.
+
+ifndef::upstream[]
+NOTE: You can install packages directly, but {org-name} recommends using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks. In addition, a `requirements.txt` file is helpful when you want to deploy a model with an S2I build.
+endif::[]
+ifdef::upstream[]
+NOTE: You can install packages directly, but using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks is recommended. In addition, a `requirements.txt` file is helpful when you want to deploy a model with an S2I build.
```

Comment:
You can install packages directly, but using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks is recommended. 
suggested edit:
Although you can install packages directly, it is recommended that you use  a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks.